control-plane/csi-driver/src/bin/controller/client.rs

```diff
@@ -408,6 +408,16 @@ impl IoEngineApiClient {
             )
             .await?;
 
-        Ok(snapshots.into_body())
+        let next_token = snapshots.body().next_token;
+        // Return only the snapshots that are Created, leaving out other statuses if any.
```

Comment:
```suggestion
        // Return only the snapshots that have been created, leaving out other statuses if any.
```

Reply:
I'd rather modify this comment now as `Don't return snapshots that are still in Creating state`

Reply:
sounds good to me